browser/sources.gni

```diff
@@ -372,6 +373,8 @@ if (ethereum_remote_client_enabled) {
 if (is_android) {
   brave_chrome_browser_sources += [
     "//brave/browser/android/brave_init_android.cc",
+    "//brave/browser/android/brave_tab_features_android.cc",
+    "//brave/browser/android/brave_tab_features_android.h",
```

Comment:
note: this is what upstream does

Reply:
That's probably because there is a circular dependency in theirs? If we can avoid adding new sources to sources.gni directly, we should

Reply:
I can put these in their own target, but I'll need to add a dep on `//chrome/browser/android` and `allow_circular_includes_from` - is that okay?